src/Middleware/src/Headstart.API/Commands/LineItemCommand.cs

```diff
@@ -96,7 +96,7 @@ public async Task<List<HSLineItem>> UpdateLineItemStatusesAndNotifyIfApplicable(
             var buyerOrder = await _oc.Orders.GetAsync<HSOrder>(OrderDirection.Incoming, buyerOrderID);
             var allLineItemsForOrder = await  _oc.LineItems.ListAllAsync<HSLineItem>(OrderDirection.Incoming, buyerOrderID);
             var lineItemsChanged = allLineItemsForOrder.Where(li => lineItemStatusChanges.Changes.Select(li => li.ID).Contains(li.ID)).ToList();
-            var supplierIDsRelatingToChange = lineItemsChanged.Select(li => li.SupplierID).Distinct().ToList();
+            var supplierIDsRelatingToChange = lineItemsChanged.Select(li => li.SupplierID).Distinct().Where(id => id != null).ToList();
```

Comment:
Do we know what happens to the value `lineItem.SupplierID` when a product is seller owned?  Does that value get populated, or is there a possibility that this value would be `null`?